### PR TITLE
OAEWDGT-107 - Add labels for summary & description

### PR DESCRIPTION
**Pull request by christianvuerings**
Jira: https://jira.sakaiproject.org/browse/OAEWDGT-107

Also updated the CSS so it looks more similar to
submit a widget page

### CODE REVIEW

File: app/views/users/registrations/_form.html.erb [hunk 1]
```diff
@@ -49,7 +49,9 @@
 
     <div class="register_column2">
         <h2>Introductory text</h2>
+        <%= f.label :info, "Basic information" %>
         <%= f.text_area :info, :placeholder => "Enter some basic intro text...", :class => "register_basic" %>
```
Comment (croby) [inline app/views/users/registrations/_form.html.erb]: Let's change these placeholders to be something more meaningful now that we have a label on the field.

Comment (christianvuerings) [inline app/views/users/registrations/_form.html.erb]: Anything specific you had in mind?

Comment (croby) [inline app/views/users/registrations/_form.html.erb]: Any sort of example of basic information and/or a quick summary I suppose. Maybe the designs have some example? If not, feel free to make something up

Comment (christianvuerings) [inline app/views/users/registrations/_form.html.erb]: When I looked up the designs, I just saw the original placeholder text.
I made up something myself.

**Pull request merged**